src/Memstate.Docs.GettingStarted/QuickStart/QuickStartTests.cs

```diff
@@ -71,11 +73,61 @@ public async Task Simple_end_to_end_getting_started_using_default_settings_with_
             await engine.DisposeAsync();
         }
 
+        // rules for using Json serialisation
+        // ----------------------------------
+        // Commands must have public empty constructors
+        // properties must have { get; set; } or at minimum { get; private set; }
+
+        [Test]
+        public async Task Simple_end_to_end_getting_started_configure_to_use_json_serializer_with_filesytem_storage()
+        {
+
+            Print("GIVEN I start a new Memstate engine for a LoyaltyDB using default settings");
+            Print($"   (using Json format & local filesystem storage)");
+
+            var config = Config.Current;
+            config.SerializerName = "NewtonSoft.Json";
+            var settings = config.GetSettings<EngineSettings>();
+            settings.StreamName = JsonFileName;
+            var engine = await Engine.Start<LoyaltyDB>();
+
```

Comment:
Auto means that it will pick the first serializer available in a specific order. But good point with the strings, we should have constants, such as `Serializers.NewtonSoftJson` and `Serializers.Wire`

Reply:
will leave it like this for now and update the tests as soon as we have them.